src/test/scala/com/workday/warp/adapters/gatling/WarpGatlingSpec.scala

```diff
@@ -67,8 +67,8 @@ object WarpGatlingSpec {
   class CustomSimulation extends WarpSimulation("MyCustomSimulationTest")
 
   // GatlingFunSpec requires `baseURL` to be implemented
```

Comment:
is this still the case? could we leave `baseUrl` abstract in WarpFunSpec?

Reply:
yeah, `baseUrl` needs to be defined for class instantiation since it's inherited from `GatlingHttpFunSpec`.